pinot-plugins/pinot-stream-ingestion/pinot-kafka-2.0/src/main/java/org/apache/pinot/plugin/stream/kafka20/KafkaConfigBackwardCompatibleUtils.java

```diff
@@ -57,9 +64,58 @@ public static void handleStreamConfig(StreamConfig streamConfig) {
           }
         }
       }
+
       if (updated) {
         entry.setValue(String.join(" ", valueParts));
       }
     }
+
+    // Read the file specified by the -Djava.security.auth.login.config VM argument
+    String loginConfigFile = System.getProperty("java.security.auth.login.config");
```

Comment:
The config file is not in a well-known format like json ?

Reply:
Here is a ref:https://docs.oracle.com/javase/8/docs/technotes/guides/security/jgss/tutorials/LoginConfigFile.html